### ISSUE TITLE
R-INLA requires rgdal

### ISSUE THREAD

**Issue by verdurin** [easybuilders/easybuild-easyconfigs#13178]
We found that `R-INLA` was missing a dependency on `rgdal`. Without that, we saw errors like this:
`Error in inla.has_PROJ6() : inla.require("rgdal") is not TRUE`

**Comment by verdurin** [comment]
Should be resolved by #13179

**Comment by branfosj** [comment]
It is in the `suggests` not the `depends` section of the R package dependencies.

```
Suggests: Deriv, Ecdat, HKprocess, MatrixModels, Rgraphviz, deldir,
        devtools, doParallel, dplyr, evd, fields, ggplot2, graph,
        gridExtra, knitr, markdown, matrixStats, mlogit, mpoly,
        mvtnorm, numDeriv, orthopolynom, pixmap, rgdal, rgeos, rgl,
        rmarkdown, shiny, sn, spdep, splancs, symmoments, testthat,
        tools
```

**Comment by verdurin** [comment]
Yes, but when it fails upon the first attempt at use, I think we need to fix that.

**Comment by branfosj** [comment]
fixed in #13179
